File: src/main/java/model/Position.java
package model;

public class Position {
    private Cell cell;
    private boolean shot;

    public Position(Cell cell) {
        this.cell = cell;
        this.shot = false;
    }

    public Cell getCell() {
        return cell;
    }

    public boolean isShot() {
        return shot;
    }

    public void setCell(Cell cell) {
        this.cell = cell;
    }

    public void setShot(boolean shot) {
        this.shot = shot;
    }
}

File: docs/scripts/Position.js
export default class Position {
    constructor(cell) {
        this.cell = cell;
        this.shot = false;
    }
    getCell() {
        return this.cell;
    }
    isShot() {
        return this.shot;
    }
    setCell(cell) {
        this.cell = cell;
    }
    setShot(shot) {
        this.shot = shot;
    }
}